### DISCUSSION TITLE
How to use QGIS with i3?

### DISCUSSION THREAD

**Discussion by jidanni** [i3/i3#5524] [Q&A]
Can someone share their i3 configuration that allows them to use QGIS? Thanks.
https://github.com/qgis/QGIS-Documentation/issues/8324

**Comment by Blenux** [comment]
You just need to add this below to your i3 config file to close any focused window. (note you only need one #, just added another due to githubs markdown thing)

##Kill focused window
bindsym $mod+Shift+q kill
